client/app/scripts/liveblog-edit/views/post.html

```diff
@@ -59,17 +59,19 @@
                 </div>
                 <div ng-if="post.multiple_items" class="lb-post__list">
                     <div ng-if="!show_all" title="Click to show all items" ng-click="toggleMultipleItems()" class="multiple-item lb-post__item">
-                        <div lb-bind-html html-content="{{ post.mainItem.item.text }}"></div>
+                        <div ng-init="item=post.items[0]">
```

Comment:
you don't have to create a div here for that. Just use `ng-init="item=post.items[0]"` in `<ng-include/>` tag